src/duration.js

```diff
@@ -864,6 +864,11 @@ export default class Duration {
 
   /**
    * Get the years.
+   *
+   * Note: Durations created out of Duration.from* (e.g. fromObject) factories will always yield 0.
+   * This will give you the correct difference in years, if the Duration was created out of a diff operation.
```

Comment:
This is factually incorrect. `Duration.fromObject({ years: 2 }).years` is `2`, not `0`.
I am not sure what you're referring to by "correct difference in years" here.